### DISCUSSION TITLE
List of models supporting docker-acap and docker-compose-acap

### DISCUSSION THREAD

**Discussion by bderaemy** [AxisCommunications/discussions#197] [ACAP] [answered]
The ability to run containers directly into the device seems promising: we tested it successfully on a device supporting it, i.e. a device which has already the _containerd_ container runtime required by the [docker-compose-acap ](https://github.com/AxisCommunications/docker-compose-acap). Now, in order to know the possible coverage of such "dockerized" ACAP, any place where we can find an updated list of which models have this  _containerd_ container runtime installed ?

**Comment by daniel-falk** [comment, marked as answer]
@bderaemy Devices listed as compatible with the Computer Vision SDK has the containerd runtime. You find the list [here](https://axiscommunications.github.io/acap-documentation/docs/axis-devices-and-compatibility/#acap-computer-vision-sdk). Remember to think of your deployment strategy before spending a lot of time on dockerized ACAPs.

![sdk4](https://user-images.githubusercontent.com/23218783/227708487-c98282d0-cac3-4892-9279-5cfbc175a102.png)

**Reply by bderaemy** [reply]
thank you @daniel-falk
